packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -1958,6 +1958,193 @@ describe("calcite-input-date-picker", () => {
   });
 });
 
+describe("proximitySelectionDisabled", () => {
+  it("should reset range and select a new start date on third click", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      html`<calcite-input-date-picker range proximity-selection-disabled min="2099-01-01"></calcite-input-date-picker>`,
+    );
+    await skipAnimations(page);
+    await page.waitForChanges();
+
+    const inputDatePicker = await page.find("calcite-input-date-picker");
+    const startInput = await page.find("calcite-input-date-picker >>> calcite-input-text");
+    const calendar = await page.find(`calcite-input-date-picker >>> .${CSS.calendarWrapper}`);
+
+    expect(await calendar.isVisible()).toBe(false);
```

Comment:
Calendar toggling is covered in other tests, so this could be dropped if not essential. Applies to other tests.

Reply:
Yes, but none of those test target `proximity-selection-disabled`. These assertions could help date-picker visibility regressions related to `proximity-selection-disabled`.